src/trio/_core/_tests/test_multierror.py

```diff
@@ -136,7 +136,7 @@ async def test_MultiErrorNotHashable() -> None:
     assert exc1 != exc2
     assert exc1 != exc3
 
-    with pytest.raises(MultiError):
+    with pytest.raises(MultiError):  # noqa: PT012
```

Comment:
PT012 is a bit of a pain when `trio` so often tests context managers, so not entirely sure it's worth having enabled. It did catch a couple ugly cases this one time though.